kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -244,3 +263,106 @@ message EmptyDirMount {
     optional string medium = 3;
     optional string size_limit = 4;
 }
+
+// The proto messages below are copied from pipeline_spec.proto

```

Comment:
Could you add a CI check to ensure this stays in sync? Also adding a comment indicating why it needs to be copied would be nice.

Reply:
okay just simply importing works now so this is no longer needed